NEWS.md

```diff
@@ -1,3 +1,14 @@
+## 1.0.3 (Released)
```

Comment:
Shouldn't there be an ` ## Unreleased` before `Released` section for eg: `## 1.1.0 - Unreleased` ?